ConnectorAutocadCivil/ConnectorAutocadCivil/UI/ConnectorBindingsAutocadCivil.Receive.cs

```diff
@@ -271,6 +284,21 @@ string id
         }
         progress.Report.Merge(converter.Report);
 
+        // track the object type counts as an event before we try to receive
+        // this will tell us the composition of a commit the user is trying to receive, even if it's not successfully received
+        // we are capped at 255 properties for mixpanel events, so we need to check dict entries
+        var typeCountArray = typeCountDict
+          .ToArray()
```

Comment:
To nitpick a little here. Is this initial `.ToArray()` needed, is it doing anything here?